src/views/SystemIntake/RequestDetails/index.tsx

```diff
@@ -1,5 +1,206 @@
 
-const RequestDetails = () => <div>Request Details</div>;
+type RequestDetailsProps = {
+  formikProps: FormikProps<SystemIntakeForm>;
+};
+
+const RequestDetails = ({ formikProps }: RequestDetailsProps) => {
+  const { values, setFieldValue } = formikProps;
+
+  return (
+    <>
+      <h2 className="font-heading-xl">Request Details</h2>
+      <p className="line-height-body-6">
+        Provide a detailed explanation of the business need/issue/problem that
+        the requested project will address, including any legislative mandates,
+        regulations, etc. Include any expected benefits from the investment of
+        organizational resources into this project. Please be sure to indicate
+        clearly any/all relevant deadlines (e.g., statutory deadlines that CMS
+        must meet). Explain the benefits of developing an IT solution for this
+        need.
+      </p>
+      <div className="grid-col-8 margin-bottom-7">
+        <Field
+          as={TextField}
+          id="IntakeForm-ProjectName"
+          label="Project Name"
+          maxLength={50}
+          name="projectName"
+        />
+
+        <fieldset className="usa-fieldset margin-top-3">
+          <legend className="usa-label margin-bottom-1">
+            Is this project funded from an existing funding source?
+          </legend>
+          <HelpText>
+            If yes, please provide your six digit funding number found [TBA this
+            is where you find it]
+          </HelpText>
+
+          <Field
+            as={RadioField}
+            checked={values.fundingSource.isFunded}
+            id="IntakeForm-HasFundingSourceYes"
+            name="fundingSource.isFunded"
+            label="Yes"
+            onChange={() => {
+              setFieldValue('fundingSource.isFunded', true);
+            }}
+            value
+          />
+          {values.fundingSource.isFunded && (
+            <div className="width-card margin-top-neg-2 margin-left-3 margin-bottom-1">
+              <Field
+                as={TextField}
+                id="IntakeForm-FundingNumber"
+                label="Funding Number"
+                maxLength={6}
+                name="fundingSource.fundingNumber"
+              />
+            </div>
+          )}
+          <Field
+            as={RadioField}
+            checked={values.fundingSource.isFunded === false}
+            id="IntakeForm-HasFundingSourceNo"
+            name="fundingSource.isFunded"
+            label="No"
+            onChange={() => {
+              setFieldValue('fundingSource.isFunded', false);
+              setFieldValue('fundingSource.fundingNumber', '');
+            }}
+            value={false}
+          />
+        </fieldset>
+
+        <Field
+          as={TextAreaField}
+          id="IntakeForm-BusinessNeed"
+          maxLength={2000}
+          label="What is your business need?"
+          helpText={
+            <p>
+              Include:
+              <br />
+              <ul>
```

Comment:
This is in the console.

```
Warning: validateDOMNesting(...): <ul> cannot appear as a descendant of <p>.
```

Reply:
Actually, after further thought, we should be more declarative instead of passing markup through props. Something like the snippet below should suffice for a `Label` component.

```
import React from 'react';
import classnames from 'classnames';

  type LabelProps = {
   children: React.ReactNode
   htmlFor: string
   className?: string
 }

  export const Label = ({children, htmlFor, className}: LabelProps) => {
    const classes = classnames(
     'usa-label',
     className
   )

    return (
     <label className={classes} htmlFor={htmlFor}>
       {children}
     </label>
   );
 }

  export default Label;
```